lib/struct.js

```diff
@@ -398,7 +395,6 @@ module.exports = function() {
         extractShort: extractShort,
         extractFloat: extractFloat,
         extractByte: extractByte,
-        extractSByte: extractSByte,
```

Comment:
Good eye! Yes, intentional - the other signed versions aren't exported either.

Added tests to `testStruct.js`, which all pass - I should've done this the first time round!